Automattic/jetpack
# Jetpack: Remove unused file

The header.php file seems to be no longer needed after https://github.com/Automattic/jetpack/pull/9952 . Opening this as a separate PR just to give an extra chance to ensure I didn't miss anything.

#### Changes proposed in this Pull Request:

* Delete header.php

#### Jetpack product discussion


n/a

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Go to the Jetpack admin, make sure everything looks good.
* Go to the old stats page, confirmed the same.
